.github/issue_template/feature.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🚀 Feature
+about: Submit a proposal for a new feature
+labels: "feature"
+---
+
+## 🚀 Feature
+
+(A clear and concise description of what the feature is.)
+
+### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+## Motivation
```

Comment:
I'm not sure I understand the difference between "Motivation" and "Pitch".

Reply:
@JelleZijlstra I have removed the Motivation section.